<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>post_item</class>
 <widget class="QWidget" name="post_item">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_name">
      <property name="text">
       <string>名字</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_time">
      <property name="text">
       <string>时间</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_join">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>参加</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_state">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>征求类型：</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>341</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_user">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_type">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>类型：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>359</x>
     <y>220</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_over">
   <property name="geometry">
    <rect>
     <x>319</x>
     <y>220</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>over</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_end">
   <property name="geometry">
    <rect>
     <x>279</x>
     <y>220</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>结束</string>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>pushButton_join</zorder>
  <zorder>textBrowser</zorder>
  <zorder>pushButton_user</zorder>
  <zorder>label_state</zorder>
  <zorder>label_type</zorder>
  <zorder>pushButton_delete</zorder>
  <zorder>pushButton_over</zorder>
  <zorder>pushButton_end</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
